packages/ember-glimmer/tests/integration/helpers/yield-test.js

```diff
@@ -0,0 +1,256 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import { set } from 'ember-metal/property_set';
+import { Component } from '../../utils/helpers';
+
+moduleFor('Helpers test: {{yield}} helper', class extends RenderingTest {
+
+  ['@test a component with a layout set renders its template where the {{yield}} helper appears']() {
+    this.registerComponent('yield-comp', { template: '<div class="wrapper">{{yield}}</div>' });
+
+    this.render('{{#yield-comp}}<div class="page-body">page body</div>{{/yield-comp}}', { title: 'test' });
+
+    equal(this.$('div.wrapper div.page-body').length, 1, 'page-body is embedded within wrapping my-page');
+
+    this.assertStableRerender();
+  }
+
+  ['@test block should work properly even when templates are not hard-coded']() {
+    this.registerComponent('outer-comp', { template: '<div class="wrapper">{{yield}}</div>' });
```

Comment:
The title of this test doesn't really make sense to me. What was it trying to test?